src/jdk.jartool/share/classes/sun/tools/jar/Main.java

```diff
@@ -582,6 +583,9 @@ boolean parseArgs(String[] args) {
                         case '0':
                             flag0 = true;
                             break;
+                        case 'k':
+                            kflag = true;
```

Comment:
I am wondering if a warning should be displayed if this options is specified with any option other than 'x'

Reply:
I considered that, but didn't implement it after confirmed other similar options didn't display any warning.
I do think it make sense to show a warning when using an option not valid in specific mode.

Reply:
OK, I will add a message. The other option I am talking about is specific mode options, like compression or manifest instead of main options.